recipes/nestly/build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install     # Python command to install the script.
```

Comment:
Please delete this file. It's functionality is covered by the `build: script:` section of the meta.yaml file